#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_security_version.patch.dpatch by Duncan Findlay <duncf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Mark this version of spamassassin with "deb" so people know
## DP: it's modified (i.e. to show it's got security fixes)

@DPATCH@
diff -urNad spamassassin~/lib/Mail/SpamAssassin.pm spamassassin/lib/Mail/SpamAssassin.pm
--- spamassassin~/lib/Mail/SpamAssassin.pm	2006-10-09 13:35:00.000000000 -0400
+++ spamassassin/lib/Mail/SpamAssassin.pm	2007-02-15 00:26:26.000000000 -0500
@@ -105,7 +105,7 @@
 
 # If you hacked up your SA, you should add a version_tag to your .cf files.
 # This variable should not be modified directly.
-@EXTRA_VERSION = qw();
+@EXTRA_VERSION = qw(deb);
 if (defined $IS_DEVEL_BUILD && $IS_DEVEL_BUILD) {
   push(@EXTRA_VERSION,
        ('r' . qw{$LastChangedRevision: 453304 $ updated by SVN}[1]));
